common.js

```diff
@@ -4,7 +4,7 @@ var config = require('./config'),
 function getFormattedDate(date, date_format) {
     date_format = date_format || config.date_format;
 
-    return moment(date).format(date_format);
+    return moment(date, date_format);
```

Comment:
actually this `date_format` is the way user wants it in the output. So, the `.format` is compulsory, we just need to change

`moment(date)` to `moment(date, DATE_FORMAT)`

`DATE_FORMAT` is the format jsonresume allows the developers to specify. Basically this function is for if a user has `2014-02-02`(first argument) as `startDate` but the theme developer wants to display this date as `02 Feb, 2014`(this is the second argument).